tests/Library/Vanilla/EmbeddedContent/Embeds/QuoteEmbedFilterTest.php

```diff
@@ -117,7 +117,7 @@ public function testFullContentFiltering() {
     class="js-embed embedResponsive"
     data-embedJson="{&quot;url&quot;:&quot;http:\/\/test.com&quot;,&quot;embedType&quot;:&quot;testEmbedType&quot;}"
 >
-    <a href="http://test.com">
+    <a href="http://test.com" rel="nofollow noreferrer ugc">
```

Comment:
This doesn't seem right.

Quote embed shouldn't have these at all (because they refer to our own site). That's part of why they have a separate view `QuoteEmbed.twig`.

Reply:
There's definitely some pre-existing issue in that test. The proper Twig view is used on the site, but not in this test. I've reverted the "fix" to expose the issue again.

Reply:
To recap: those attribute values _should_ be there, because we're evaluating the inner quote, not the outer quote. We'd expect the other view to be used for the outer quote. The inner quote is purely the quotes internal content.